src/ControlsTest/BasicTests.cs

```diff
@@ -0,0 +1,49 @@
+﻿using Esri.ArcGISRuntime.OpenSourceApps.DataCollection.CustomControls;
+using NUnit.Framework;
+using System.Windows;
+using System.Windows.Controls;
+
+namespace ControlsTest
+{
+    [TestFixture]
+    public class BasicTests
+    {
+        [TestCase(0,0)]
+        [TestCase(300,300)]
+        [TestCase(0,600)]
+        [TestCase(600,0)]
+        public void NoChildren(double width, double height)
+        {
```

Comment:
I think both of these tests could use a comment clarifying what they are testing.